billy-portugal/src/main/java/com/premiumminds/billy/portugal/persistence/dao/jpa/DAOPTReceiptInvoiceImpl.java

```diff
@@ -53,13 +52,13 @@ protected Class<JPAPTReceiptInvoiceEntity> getEntityClass() {
     }
 
     @Override
-    public List<PTReceiptInvoiceEntity> getBusinessReceiptInvoicesForSAFTPT(StringID<Business> uid, Date from, Date to) {
+    public List<PTReceiptInvoiceEntity> getBusinessReceiptInvoicesForSAFTPT(StringID<Business> uid, LocalDate from, LocalDate to) {
         QJPAPTReceiptInvoiceEntity invoice = QJPAPTReceiptInvoiceEntity.jPAPTReceiptInvoiceEntity;
 
         JPAQuery<PTReceiptInvoiceEntity> query = this.createQuery();
 
         query.from(invoice)
-                .where(invoice.instanceOf(JPAPTReceiptInvoiceEntity.class).and(invoice.date.between(from, to))
+                .where(invoice.instanceOf(JPAPTReceiptInvoiceEntity.class).and(invoice.localDate.between(from, to))
```

Comment:
Just a reminder to ensure the release notes include this constraint regarding SAFT